pkgs/shells/zsh/zimfw/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zimfw";
+  version = "v1.11.0";
+  src = fetchFromGitHub {
+    owner = "zimfw";
+    repo = pname;
+    rev = version;
```

```suggestion
  pname = "zimfw";
  version = "1.11.0";
  src = fetchFromGitHub {
    owner = "zimfw";
    repo = "zimfw";
    rev = "v${version}";
```

Using `repo = pname` is now discouraged afaik and the version should be just the version number without the prefix